app/assets/stylesheets/mastodon.css.scss

```diff
@@ -0,0 +1,45 @@
+@font-face {
```

Comment:
Is this files copied from somewhere? 🤔

Reply:
I took the main parts from here:

https://github.com/FortAwesome/Font-Awesome/issues/10897#issuecomment-318878737

I just adjusted some paths.